locale/en/manager.po

```diff
@@ -2254,6 +2254,15 @@ msgstr ""
 "The submission file {$id} was skipped because it is attached to a record "
 "that will not be imported, such as a review assignment or discussion."
 
+msgid "plugins.importexport.native.error.submissionFileWithoutRevision"
+msgstr "The submission file {$id} was skipped because it does not have a valid revision."
```

Comment:
Is this a sanity check, or this change could produce cases that the revision is not valid?

I am wondering whether skipping revised files could result a "break" in the "revision chain" which may result to inconsistences